templates/media-add-list.html.twig

```diff
@@ -17,12 +17,16 @@
  */
 #}
 {% if bundles is not empty %}
-    <dl>
+    <ul class="admin-list">
```

Comment:
This template is based on node's template for same page. Why do we need to change it?